src/client/pps/pps.proto

```diff
@@ -342,6 +360,15 @@ message RestartDatumRequest {
   repeated string data_filters = 2;
 }
 
+message InspectDatumRequest {
+  string datum_id = 1 [(gogoproto.customname) = "DatumID"];
+  string job_id = 2 [(gogoproto.customname) = "JobID"];
+}
```

Comment:
Ok, this should be done now. The existing `Datum` message was renamed `InputFile`